l10n_it_fatturapa_in_rc/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     "name": "ITA - Fattura elettronica - Inversione contabile",
     "summary": "Modulo ponte tra e-fattura in acquisto e inversione" " contabile",
-    "version": "14.0.1.1.1",
+    "version": "16.0.1.0.0",
```

Comment:
Questa modifica dovrebbe essere nel commit di migrazione, non in quello di `pre-commit`.
Per aiutarti con il processo di migrazione puoi usare https://github.com/OCA/odoo-module-migrator

Reply:
Fatto